### PR TITLE
Improve Vtodo support / add Maker

### PR DESCRIPTION
**Pull request by dpocock**
Please let me know if this is OK

I notice some unit tests fail on the master branch even before I made any changes.  The same unit tests fail after this change so I don't think I have broken anything.

Have you thought about using travis-ci to make builds of this repository?

### CODE REVIEW

File: lib/vpim/vtodo.rb [hunk 1]
```diff
@@ -97,6 +97,52 @@ def percent_complete
...
+          yield m
+          m.todo
+        end
+
+        # Set transparency to "OPAQUE" or "TRANSPARENT", see Vpim::Vtodo#transparency.
+        def transparency(token)
+          set_token 'TRANSP', ["OPAQUE", "TRANSPARENT"], "OPAQUE", token
+        end
+
+        # Add a RRULE to this todo. The rule can be provided as a pre-build
```
Comment (sam-github) [inline lib/vpim/vtodo.rb]: "pre-built"

**Comment by sam-github** [comment]
Tiny spelling error, but otherwise looks great.

About travis, I'm not looking for more work to do, but I'm happy to accept a PR.

**Comment by dpocock** [comment]
I cut and pasted that block of code (typo and all) from Vevent.  I've added a commit that fixes the typos.

**Comment by sam-github** [comment]
:blush: and thank you.

**Comment by dpocock** [comment]
We are going to use Vpim for the Debian Maintainer Dashboard:

https://udd.debian.org/dmd/

Do you think you could make another tag once I finish testing Vtodo changes?

Here is what I've done so far for DMD:

http://anonscm.debian.org/cgit/collab-qa/udd.git/commit/?id=e8901fd530b344b4b6a24fe0c78a222709417d7d

**Comment by sam-github** [comment]
I added you as a collaborator, feel free to make the tag.

Other changes, please PR them for review. Thanks.

**Comment by dpocock** [comment]
Ok, thanks, I've sent the PR for travis-ci.  Would you mind if I push changes to that one file as well if necessary to make it build?

**Comment by sam-github** [comment]
No problem, push away.​

**Pull request merged**